node/src/main/scala/com/wavesplatform/block/Block.scala

```diff
@@ -47,6 +48,66 @@ class BlockHeader(val timestamp: Long,
 }
 
 object BlockHeader extends ScorexLogging {
+  def writeHeaderOnly(bh: BlockHeader): Array[Byte] = {
+    val ndo = ByteStreams.newDataOutput()
+
+    ndo.writeByte(bh.version)
+    ndo.writeLong(bh.timestamp)
+    ndo.write(bh.reference)
+    ndo.writeLong(bh.consensusData.baseTarget)
+    ndo.write(bh.consensusData.generationSignature)
+
+    if (bh.version == 1 | bh.version == 2)
+      ndo.writeByte(bh.transactionCount)
+    else
+      ndo.writeInt(bh.transactionCount)
+
+    ndo.writeInt(bh.featureVotes.size)
+    bh.featureVotes.foreach(s => ndo.writeShort(s))
+    ndo.write(bh.signerData.generator)
+    ndo.write(bh.signerData.signature)
+
+    ndo.toByteArray
+  }
+
+  def readHeaderOnly(bytes: Array[Byte]): BlockHeader = {
```

Comment:
Why not reuse `com.wavesplatform.block.BlockHeader#parseBytes`?

Reply:
Because it parses not only blockheader bytes but also transactions